.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  parser: "@typescript-eslint/parser", // Specifies the ESLint parser
+  extends: [
+    "plugin:react/recommended", // Uses the recommended rules from @eslint-plugin-react
+    "plugin:@typescript-eslint/recommended", // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+    "prettier/@typescript-eslint", // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    "plugin:prettier/recommended" // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+  ],
+  parserOptions: {
+    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+    sourceType: "module" // Allows for the use of imports
+  },
+  "rules": {
+	"react/react-in-jsx-scope": "off"
+  },
+  "globals": {
+	"React": "writable"
+  },
+    "settings" : {
+        "react": {
+            "version": "detect"
+        }
+    }
+};
```

Comment:
We have a quite similar setup for most of our other repos, maybe just copy that? https://github.com/dotkom/glowing-fortnight/blob/master/.eslintrc.js

Reply:
Copied!